src/types/projectTypes.ts

```diff
@@ -33,7 +30,7 @@ export interface Data {
 }
 
 export interface ProjectResponse {
```

Comment:
I thought there would now also be a `CurrentProjectResponse` to account for the return of both message handlers no?

Reply:
So just below this is the CurrentProjectResponse. There are now both types as they represent different responses from the backend. Maybe this naming convention is not working. What do you think?

Reply:
no, sorry, it was just hidden by the changes preview, my bad! This is adding up yeah